source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -180,81 +170,71 @@ Note:
 ## Exporting a VM or Template as an OVA
 The user will be able to select a VM (or a template) to be exported as
 an OVA file. In such cases, the user needs to specify a path on a host
-where the OVA file will be created.
+or a disk on shared storage where the OVA file will be created.
```

Comment:
right right, I wasn't suggesting to focus only on the local filesystem on the host. my point was whether we treat shared storage as yet another path on the host - and by that assume that it was already mounted to that path. because otherwise we would need to make a relatively big change in the import dialog (for getting the details, including the authentication details, of the NFS share) and that is something we didn't plan to do.